src/SOS/SOS.Extensions/HostServices.cs

```diff
@@ -188,7 +194,7 @@ private HostServices()
                 Trace.TraceError(ex.ToString());
                 return HResult.E_FAIL;
             }
-            if (DebuggerServices.GetSymbolPath(out string symbolPath) == HResult.S_OK)
+            if (DebuggerServices.GetSymbolPath(out string symbolPath) >= HResult.S_OK)
```

Comment:
Some of the dbgeng API's do return S_FALSE when the name or path returned has been truncated because the incoming buffer was too small.  I just wanted to make sure it succeeded with the truncated symbol path.  

Thinking about it now, I'm not sure what best thing is to do here and because DebuggerServices.GetSymbolPath queries the sympath length first and then allocates the right size buffer, S_FALSE should never be returned.  The DebuggerServices.GetSymbolPath code also checks with hr >= S_OK too when calling the native function.

Reply:
I changed it back to == S_OK and changed DebuggerServices.GetSymbolPath to also treat S_FALSE as an error.